tfhe/src/integer/server_key/radix_parallel/tests.rs

```diff
@@ -328,9 +350,16 @@ fn integer_default_add_work_efficient<P>(param: P)
 where
     P: Into<PBSParameters>,
 {
-    let (cks, sks) = KEY_CACHE.get_from_params(param);
+    let (cks, mut sks) = KEY_CACHE.get_from_params(param);
     let cks = RadixClientKey::from((cks, NB_CTXT));
 
+    let sks = if !sks.deterministic_pbs_execution() {
+        sks.set_deterministic_pbs_execution(true);
+        sks
+    } else {
+        sks
+    };
```

Comment:
There  is no need for the if I think we would just do `sks.set_deterministic_pbs_execution(true);`

Reply:
ah right we are in the default tests 🤦‍♂️